docs/activities.md

```diff
@@ -129,7 +129,7 @@ This timeout has two primary use cases:
 
 **The default Schedule-To-Start Timeout is ∞ (infinity).**
 
-If this timeout is used, we recommend setting this timeout to the maximum time a Workflow Execution is willing to wait for an Activity Execution in the presence of all possible Worker outages, and have a concrete plan in place to reroute Activity Tasks to a different Task Queue.
+If this timeout is used, Temporal recommends setting this timeout to the maximum time a Workflow Execution is willing to wait for an Activity Execution in the presence of all possible Worker outages, and have a concrete plan in place to reroute Activity Tasks to a different Task Queue.
```

Comment:
I recommend stetting "we recommend" per "should, should be" in https://developers.google.com/style/word-list?hl=en#letter-s. See also the numerous instances of "we recommend" used throughout the Google guide and the June 14, 2022, thread in #topic-documentation.

Reply:
Oops for me too.